src/chat/__tests__/Inbox.test.tsx

```diff
@@ -15,6 +18,14 @@ describe('<Inbox />', () => {
   const InboxWithProviders = withProviders(Inbox, MemoryRouter);
   beforeEach(() => {
     jest.clearAllMocks();
+    //@ts-ignore
+    mockGetInboxesList.mockResolvedValueOnce({
+      data: { results: chatConversationsMock },
+    });
+    //@ts-ignore
+    mockGetChatAuthorizationToken.mockResolvedValueOnce({
+      data: { token: 'test' },
+    });
```

Comment:
Can you provide some helpful comment why TS is ignored here and how we can fix or prevent this in the future?

Reply:
I removed ts-ignore by marking these functions as a jest.Mock :)